templates/zerver/integrations/index.html

```diff
@@ -139,7 +139,6 @@ <h3 class="integration-name create-your-own">{% trans %}Create your own!{% endtr
                 <div id="integration-instructions-group">
                     <div id="integration-instruction-block" class="integration-instruction-block">
                         <a href="/integrations" id="integration-list-link" class="no-underline"><i class="fa fa-arrow-circle-left" aria-hidden="true"></i><span>Back to list</span></a>
-                        <h3 class="name">Name</h3>
```

Comment:
Shouldn't this change be in the first commit (about integrations content modification)?

Reply:
No, because it is part of the header. The other changes are related to the content of the pages. If I add this to the first commit and then someone reverts that commit, the description page of one integration will not have a title anymore.